src/components/RelatedVideoItem/RelatedVideoItem.component.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Image, GridItem, Title } from './RelatedVideoItem.styles';
+
+const RelatedVideoItem = ({ video, videos }) => {
+  const history = useHistory();
+  const { snippet } = video;
```

Comment:
you could destructure video more so that code below result cleaner
```suggestion
  const { snippet: { title, description, thumbnails } } = video;
```